beestation.dme

```diff
@@ -2826,6 +2826,9 @@
 #include "code\modules\ruins\spaceruin_code\whiteshipruin_box.dm"
 #include "code\modules\security_levels\keycard_authentication.dm"
 #include "code\modules\security_levels\security_levels.dm"
+#include "code\modules\SCPmodular\scps\euclid\SCP-173.dm"
+#include "code\modules\SCPmodular\scps\euclid\SCP-294.dm"
+#include "code\modules\SCPmodular\scps\safe\scp999.dm"
```

Comment:
Bad structure, all the DM files should just be in a single `scp` module.

Reply:
Will fix by the next PR